src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -203,6 +203,7 @@ public void onPause() {
   }
 
   public void onNewIntent() {
+    loadChatlistAsync();
```

Comment:
this will force an update of the chatlist on every return to the chatlist, not only on tapping notifications.

we try to avoid that for performance reasons - and instead keep the list  updated in hidden state on events - that results in an incredible fast returning where no redraw is needed.

therefore, if really needed, we should force the update more selectively, eg using `RELOAD_LIST` flag

Reply:
```suggestion
```